sdk/script/account-manager.ts

```diff
@@ -45,6 +45,19 @@ interface JsonResponse {
     body?: any;
 }
 
+// A template string tag function that URL encodes the substituted values
+function urlEncode(strings: string[], ...values: string[]): string {
+    var result = "";
+    for (var i = 0; i < strings.length; i++) {
+        result += strings[i];   // Assumes that template string will not start with a substitution
```

Comment:
Oh!! Boy am I glad to hear that. I was feeling like this language feature was badly designed, but what you just explained makes perfect sense.